lighthouse-core/gather/gather-runner.js

```diff
@@ -456,14 +456,16 @@ class GatherRunner {
 
     const {emulatedFormFactor} = options.settings;
     // Whether Lighthouse was run on a mobile device (i.e. not on a desktop machine).
-    const IsMobileHost = hostUserAgent.includes('Android') || hostUserAgent.includes('Mobile');
+    const HostFormFactor = hostUserAgent.includes('Android') || hostUserAgent.includes('Mobile') ?
+      'mobile' : 'desktop';
     const TestedAsMobileDevice = emulatedFormFactor === 'mobile' ||
```

Comment:
this seems like a good thing to throw into our smoke tests now, do we have any that test as desktop? shouldn't be too bad to add

Reply:
but this is the `hostUserAgent`. we don't run smoke tests against real mobile devices :P

Reply:
I know that, notice how this comment is on the `TestedAsMobileDevice` line :P

```
HostFormFactor: 'desktop',
TestedAsMobileDevice: true,
```

seems like a reasonable thing to have in our artifacts smoketests